build.sbt

```diff
@@ -12,14 +12,15 @@ val silencerV_2_12 = "1.6.0"
 val silencerV = "1.7.0"
 
 val flink114V = "1.14.5"
-val currentFlinkV = "1.16.0"
+val flink116V = "1.16.0"
+val currentFlinkV = "1.18.1"
 val sttpV = "3.8.11"
 val kafkaV = "3.3.1"
 val testContainersScalaV = "0.41.0"
 
 ThisBuild / version := "0.1-SNAPSHOT"
 
-val defaultNussknackerV = "1.14.0"
+val defaultNussknackerV = "1.15.0-staging-2024-04-04-17639-3e42b8064-SNAPSHOT"
```

Comment:
Let's leave a comment that we should regularly bump this version and TODO that we should publish maven artifacts with x.y.z-SNAPSHOT version in the core nu project